build.sbt

```
@@ -72,43 +67,53 @@ def freshProject(name: String, dir: File): Project = {
 // Fork each scala test for now, to work around persistent mutable state
 // in Rocket-Chip based generators
 def isolateAllTests(tests: Seq[TestDefinition]) = tests map { test =>
-      val options = ForkOptions()
-      new Group(test.name, Seq(test), SubProcess(options))
-  } toSeq
+  val options = ForkOptions()
+  new Group(test.name, Seq(test), SubProcess(options))
+} toSeq
 
 // Subproject definitions begin
-//
-// FIRRTL is handled as an unmanaged dependency. Make will build the firrtl jar
-// before launching sbt if any of the firrtl source files has been updated
-// The jar is dropped in chipyard's lib/ directory, which is used as the unmanagedBase
-// for all subprojects
-lazy val chisel  = (project in file("tools/chisel3"))
+
+val chiselVersion = "3.4.0"
+lazy val chisel = (project in file("tools/chisel3"))
```

```suggestion
lazy val chisel = ProjectRef(workspaceDirectory / "chisel3", "chisel")
```
@abejgonzalez try doing exactly what RocketChip does here.

This should work with your setting of `-Dsbt.workspace=$PWD/tools` as you do in `.sbtopts`.

---

Yes. I found the issue here. It was due to FireSim referencing the `chisel` project in Chipyard. However, if you make it a `ProjectRef` the project doesn't exist in Chipyard (just in the `tools/chisel3/`). I'll push up a change that addresses this soon.